ansible_base/feature_flags/utils.py

```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+
+
+def get_django_flags():
+    if hasattr(settings, 'FLAGS'):
+        return settings.FLAGS
+    return {}
```

Comment:
I suggest using `getattr` with a default here, historically there were [problems](https://hynek.me/articles/hasattr/) with hasattr, and getattr is more efficient anyway.

```suggestion
    return getattr(settings, 'FLAGS', {})
```

Reply:
Great note! Will update